ContactManager2/src/components/ErrorMessage.jsx

```diff
@@ -0,0 +1,10 @@
+import PropTypes from "prop-types"
+
+export const ErrorMessage = (props) => {
+  const message = props.message;
+  return message != null ? <div className="error">{message}</div> : <div className="dummyHeight" />
```

Comment:
```suggestion
  return Boolean(message) ? <div className="error">{message}</div> : <div className="dummyHeight" />
```

Reply:
and why do you need add dummyHeight class here... simple add padding when we get any error

Reply:
refactored the code, removed dummyHeight.